### ISSUE TITLE
Remove close button from popup-notifications

### ISSUE THREAD

**Issue by hhhorb** [mate-desktop/mate-notification-daemon#97]
It's pointless, clicking anywhere on the notification window closes it (plus normal notifications timeout in just a few seconds anyway, a feature request to set the timeout length would be nice) .

I tried to just edit the notification theme I'm using (standard), but it looks like the 4 available themes are hardcoded into the daemon itself? That's a weird way to implement themes, shouldn't they be separated?

Anyway, kudos to the MATE team. You've put together a very well integrated DE that's nearly as light on RAM usage as Xfce is. Cheers!

https://forums.bunsenlabs.org/viewtopic.php?pid=15789#p15789
##

**Comment by raveit65** [comment]
yeah, this will save us from GtkStock deprecated warnings for gtk3 too.

**Comment by raveit65** [comment]
should be possible to add a timer gsettings key, mate-applet-streamer dev added this already.
https://github.com/assen-totin/mate-applet-streamer/commit/651ca980f3aca0b7e7a8fd5228eefd4e534e0909

**Comment by hhhorb** [comment]
Hey, nice!

**Comment by raveit65** [comment]
I noticed there is a different between using the close-button or clicking on the notification itself.
If i got an email notification from claws-mail clicking on the notification show me claws-mail window.
I have claws-mail running on a another workspace. This means clicking on the notification will switch the workspace, and this is mostly not what i want.

**Comment by waddon1** [comment]
The issue @raveit65 brought up is still an issue.

**Comment by luigifab** [comment]
Maybe when action when clicking anywhere on the notification window isn't set => hide close button.

**Comment by hhhorb** [comment]
^ Wait, what? Hey, it's your notification, you can do whatever, but what? "The difference between clicking the body and clicking the close button is valid..." Huh? Dude's, one close action. It can't be secondary closed, what the...?

**Comment by vkareh** [comment]
Clicking the close button should close the notification, right? But clicking the body can have different actions attached to it (like opening a mail client, focusing on the application that sent the notification, etc.)

So if we remove the close button as "unnecessary" then notifications that perform an action on click won't have a way of manually closing. This therefore becomes a regression bug.

Note that according to the freedesktop notification spec, not all notifications should close on timeout. So an urgent application notification, for example, would stay open forever without any way of closing or dismissing it.
